spec/api-process-spec.js

```diff
@@ -67,4 +71,18 @@ describe('process module', () => {
       expect(heapStats.doesZapGarbage).to.be.a('boolean')
     })
   })
+
+  describe('process.takeHeapSnapshot()', () => {
+    it('returns true on success', (done) => {
+      const filePath = path.join(remote.app.getPath('userData'), 'test.heapsnapshot')
+      const success = process.takeHeapSnapshot(filePath)
+      expect(success).to.be.true()
+      fs.unlink(filePath, () => done())
```

Comment:
Please call `expect` after the unlink.
And it would be better to call it sync version `fs.unlinkSync()`

Reply:
refactored a bit differently